tests/zendesk/test_zendesk.py

```diff
@@ -0,0 +1,94 @@
+import pytest
+import dlt
+from dlt.pipeline.pipeline import Pipeline
+from pipelines.zendesk.zendesk import zendesk, zendesk_chat, zendesk_talk
+from tests.utils import ALL_DESTINATIONS, assert_load_info
+
+
+# list expected tables and the number of columns they are supposed to have
+SUPPORT_TABLES = ["ticket_fields", "tickets", "ticket_metric_events", "users", "sla_policies", "groups", "organizations", "brands", "activities", "automations", "custom_agent_roles", "dynamic_content",
+                  "group_memberships", "job_status", "macros", "organization_fields", "organization_memberships", "recipient_addresses", "requests", "satisfaction_ratings", "sharing_agreements", "skips",
+                  "suspended_tickets", "targets", "ticket_forms", "ticket_metrics", "triggers", "user_fields", "views", "tags"]
+CHAT_TABLES = ["chats"]
+TALK_TABLES = ["addresses", "agents_activity", "current_queue_activity", "greeting_categories", "greetings", "ivrs", "lines", "phone_numbers", "settings"]
+ALL_DESTINATIONS = ["postgres"]
+
+
+def create_pipeline(destination_name, dataset_name, full_refresh=True, include_support: bool = False, include_chat: bool = False, include_talk: bool = False):
```

Comment:
this is good starting point. we must however test
1. for each incremental table that it loads incrementally. that could be done by inspecting state after running if the correct timestamp is there. and then running again to observe no new records
2. you must make sure that pivoting works